### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by nevisende**
We created a tag to show the error message and We write functions in JS to check email validation.

### CODE REVIEW

**Comment by D4nielJ** [comment]
Hi @nevisende, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't check for all of the linter requirements. Please update the linters of your project so they checked for JavaScript code as well. To do this remove the old ones and follow the instructions to [install HTML/CSS/JavaScript Linters.](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -173,14 +174,43 @@ <h2>Get Started</h2>
                            <input type="email" name="emial" id="email" required>
                      </label>
                  </div>
-    
                  <textarea name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Enter your message here..." required></textarea>
                  <button type="submit"> Start collaboration</button>
```
Comment (taiwo2) [inline index.html:178]: The error message should fit the design from the chosen template in Figma (colors). the color chosen should fit the design template. kindly fix it

**Review by taiwo2 — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -173,14 +174,43 @@ <h2>Get Started</h2>
                            <input type="email" name="emial" id="email" required>
```
Comment (taiwo2) [inline index.html:174]: KIndly correct the spelling of your name attribute

Comment (taiwo2) [inline index.html:174]: KIndly correct the spelling of your name attribute

**Review by taiwo2 — CHANGES_REQUESTED** [review]

**Review by ignatius22 — APPROVED** [review]
## Hi @nevisende, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**